src/components/core/PivotTable.tsx

```diff
@@ -494,18 +501,20 @@ export class PivotTableInner extends
     }
 
     public cellClicked = (cellEvent: IGridCellEvent) => {
-        const { drillableItems, onFiredDrillEvent } = this.props;
+        const { onFiredDrillEvent } = this.props;
         const { columnDefs } = this.state;
         const afm: AFM.IAfm = this.props.dataSource.getAfm();
+        const drillablePredicates = this.getDrillablePredicates();
 
         const { colDef, rowIndex } = cellEvent;
         const isRowTotal = get<IGridCellEvent, string>(cellEvent, ['data', 'type', ROW_TOTAL]);
-        const rowDrillItem = get<IGridCellEvent, IDrillableItem>(cellEvent, ['data', 'drillItemMap', colDef.field]);
-        const drillItems = rowDrillItem ? [...colDef.drillItems, rowDrillItem] : colDef.drillItems;
+        const rowDrillItem = get<IGridCellEvent, IMappingHeader>(cellEvent, ['data', 'drillItemMap', colDef.field]);
+        const drillItems: IMappingHeader[] = rowDrillItem ? [...colDef.drillItems, rowDrillItem] : colDef.drillItems;
```

Comment:
keep same naming if possible as, e.g. in `getCellClass`, line 422

Reply:
Sorry, I don't understand.